### PR TITLE
Don't attempt to create data if the event's fileURL is nil.

### LINKED ISSUE #5082: Crash in GoogleDataTransportCCTSupport v2.0
### [REQUIRED] Step 1: Describe your environment

  * Xcode version: 11.3.1
  * Firebase SDK version: 6.19.0
  * Firebase Component: GoogleDataTransportCCTSupport (Auth, Core, Database, Firestore, Messaging, Storage, etc)
  * Component version: 2.0
  * Installation method: CocoaPods

#### Steps to reproduce:

I got a crash report from a user where GoogleDataTransportCCTSupport causes an infinite crash loop during the start of the app. Cannot reproduce by myself.

#### Relevant Code:

```
Last Exception Backtrace:
0   CoreFoundation                	0x1adc72a48 __exceptionPreprocess + 220
1   libobjc.A.dylib               	0x1ad999fa4 objc_exception_throw + 55
2   Foundation                    	0x1adf30054 -[NSData+ 53332 (NSData) initWithContentsOfURL:options:maxLength:error:] + 631
3   Foundation                    	0x1adf5b5b4 +[NSData+ 230836 (NSData) dataWithContentsOfURL:options:error:] + 67
4   AppName                         	0x104ce5460 GDTCCTConstructLogEvent + 2495584 (GDTCCTNanopbHelpers.m:0)
5   AppName                         	0x104ce4fe8 GDTCCTConstructLogRequest + 2494440 (GDTCCTNanopbHelpers.m:120)
6   AppName                         	0x104ce4dfc __GDTCCTConstructBatchedLogRequest_block_invoke + 2493948 (GDTCCTNanopbHelpers.m:91)
7   CoreFoundation                	0x1adcbfd74 __NSDICTIONARY_IS_CALLING_OUT_TO_A_BLOCK__ + 15
8   CoreFoundation                	0x1adb497dc -[__NSDictionaryM enumerateKeysAndObjectsWithOptions:usingBlock:] + 207
9   AppName                         	0x104ce4d38 GDTCCTConstructBatchedLogRequest + 2493752 (GDTCCTNanopbHelpers.m:87)
10  AppName                         	0x104ce7ec4 -[GDTCCTUploader constructRequestProtoFromPackage:] + 2506436 (GDTCCTUploader.m:358)
11  AppName                         	0x104ce77ac __32-[GDTCCTUploader uploadPackage:]_block_invoke.51 + 2504620 (GDTCCTUploader.m:273)
12  libdispatch.dylib             	0x1ad93e610 _dispatch_call_block_and_release + 23
13  libdispatch.dylib             	0x1ad93f184 _dispatch_client_callout + 15
14  libdispatch.dylib             	0x1ad91c710 _dispatch_lane_serial_drain$VARIANT$armv81 + 563
15  libdispatch.dylib             	0x1ad91d128 _dispatch_lane_invoke$VARIANT$armv81 + 399
16  libdispatch.dylib             	0x1ad92643c _dispatch_workloop_worker_thread + 575
17  libsystem_pthread.dylib       	0x1ad98eb88 _pthread_wqthread + 275
18  libsystem_pthread.dylib       	0x1ad991760 start_wqthread + 7
```

### PR DESCRIPTION
**Pull request by mikehaney24**
This could be because of changes to NSCoding w.r.t. transitioning from GDTCORStoredEvent to GDTCOREvent.

Fixes #5082

### CODE REVIEW

File: GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTNanopbHelpers.m [hunk 1]
```diff
@@ -148,7 +148,13 @@ gdt_cct_LogEvent GDTCCTConstructLogEvent(GDTCOREvent *event) {
     logEvent.has_network_connection_info = 1;
   }
   NSError *error;
-  NSData *extensionBytes = [NSData dataWithContentsOfURL:event.fileURL options:0 error:&error];
+  NSData *extensionBytes;
+  if (event.fileURL) {
+    extensionBytes = [NSData dataWithContentsOfURL:event.fileURL options:0 error:&error];
+  } else {
+    GDTCORLogError(GDTCORMCEFileWriteError, @"%@", @"An event's fileURL property was nil.");
```
Comment (tfonfara) [inline GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTNanopbHelpers.m]: Should be `GDTCORMCEFileReadError ` I guess

Comment (mikehaney24) [reply GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTNanopbHelpers.m]: Of course, you're right. Ty.

**Comment by wangjunhua1** [comment]
I met the same problem today, how to solve it

**Comment by wangjunhua1** [comment]
I met the same problem today. When will a new version be available

**Comment by mikehaney24** [comment]
Hopefully we'll release a new SDK version this week, but it's always possible for it to be delayed.

**Comment by wangjunhua1** [comment]
> Hopefully we'll release a new SDK version this week, but it's always possible for it to be delayed.

Which version do we need to backtrack to if we want to solve it

**Comment by paulb777** [comment]
The fix is released. `pod update` should resolve.

**Comment by wangjunhua1** [comment]
> The fix is released. `pod update` should resolve.

Thank you very much

**Pull request merged**